madmom/ml/nn/layers.py

```diff
@@ -309,38 +316,46 @@ def activate(self, data):
             Activations for this data.
 
         """
+        # init previous output
+        if not hasattr(self, 'prev_output') or self.prev_output is None:
```

Comment:
Yes, otherwise we can not load the old models any more, since they don't have these attributes.